spec/requests/statuses/embed_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Status embed' do
+  describe 'GET /users/:account_username/statuses/:id/embed' do
+    subject { get embed_account_status_path(account.username, status) }
```

Comment:
I think we may want to either change the description, or hardcode the path (I have a slight preference for the latter). As is, the two can get out of sync without that being noticed.

Reply:
Point taken on possible divergence... in an ideal world we'd have sufficient routing specs to be confident in using these in the request specs, but short of that - fine to use the full paths here. Updated to use that style.